packages/slate/src/models/node.js

```diff
@@ -1024,9 +1003,13 @@ class Node {
    */
 
   getMarksAsArray() {
-    return this.nodes.reduce((marks, node) => {
-      return marks.concat(node.getMarksAsArray())
-    }, [])
+    // PREF: use only one concat rather than multiple concat
```

Comment:
There's a typo in all these comments, it should be `PERF:` (short for performance, though I realize this isn't the clearest) instead of `PREF:`.

Reply:
Fixed this and other typos~